TTS/bin/synthesize.py

```diff
@@ -161,6 +161,10 @@ def main():
         default=None,
     )
     parser.add_argument("--gst_style", help="Wav path file for GST stylereference.", default=None)
+    parser.add_argument(
```

Comment:
we can make use of the `style_wav` argument for capacitron rather than adding a new argument.

Reply:
do you mean merging gst style_wav and capacitron_reference wav?

Reply:
yes. I think they serve the same purpose. I might be wrong .

Reply:
you're right, they're the same. should I rename it to `style_reference_mel` and merge the two?

Reply:
but it is not mel, the input is a wav file, isn't it? `ref_wav` makes sense to me